packages/auth/lib/Roles.js

```diff
@@ -96,6 +96,20 @@ const userIsMeOrProfileVisible = (user, me) => (
   )
 )
 
+const {
+  SELF_CLAIMABLE_ROLES
+} = process.env
+
+const isRoleClaimableByMe = (role, me) => (
+  me &&
+  role &&
+  SELF_CLAIMABLE_ROLES &&
+  SELF_CLAIMABLE_ROLES.length &&
+  SELF_CLAIMABLE_ROLES
+    .split(',')
```

Comment:
As in a previous request, it is likely less performant to split `SELF_CLAIMABLE_ROLES` on each fn call.

Reply:
I count the easier readability is much higher compared to the very small performance impact.